admin_manual/installation/apps_management_installation.rst

```diff
@@ -71,17 +71,60 @@ The configuration example below shows how to add a second directory, called ``ap
    :emphasize-lines: 9-13
 
 After you add a new directory configuration, you can then move apps from the original app directory to the new one. 
-To do so, follow these steps:
+If you want to identify as an example all non core / non shipped apps, use the following command:
```

Comment:
I suggest the following to aid in clarity of intent:

If you want to identify, as an example, all non-core or non-shipped apps, use the following command: